app/Http/Controllers/Api/SearchTitleController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Post;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Tools\ResponseTool;
+
+class SearchTitleController extends Controller
+{
+    public function getPostAll()
+    {
+        return ResponseTool::success(Post::orderBy('created_at', 'desc')
```

Comment:
Make the controller simple ever, so if there is bussiness logic or custom query, you have 2 options, using scope or if it even more complicated you should put it in `Repository` layer